### PR TITLE
Add XonneX/adventofcode2020 to PHP

### PR DESCRIPTION
**Pull request by XonneX**

### CODE REVIEW

**Review by jclamar08 — COMMENTED** [review]
any

File: README.md [hunk 1]
```diff
@@ -282,6 +282,8 @@ Read [CONTRIBUTING.md](/CONTRIBUTING.md) to learn how to add your own repos.
 
 *Solutions to AoC in PHP.*
 
+* [XonneX/adventofcode2020]
```
Comment (adaamz) [inline README.md:285]:
```suggestion
* [XonneX/adventofcode2020](https://github.com/XonneX/adventofcode2020)
```

**Review by adaamz — CHANGES_REQUESTED** [review]
pls check your changes before creating PR
https://github.com/Bogdanp/awesome-advent-of-code/blob/master/README.md

File: README.md (hunk 1 above)
Comment (XonneX) [reply README.md:285]: Shouldn't we follow the instructions? https://github.com/Bogdanp/awesome-advent-of-code/blob/master/CONTRIBUTING.md
Don't want to break the GitHub Action.
If the instructions are outdated I can take this in the PR as well.

Comment (Bogdanp) [reply README.md:285]: The instructions are correct and so is your PR.

**Pull request merged**